ingestion/src/metadata/ingestion/source/database/common_db_source.py

```diff
@@ -131,9 +131,12 @@ def get_database_names(self) -> Iterable[str]:
         """
         custom_database_name = self.service_connection.__dict__.get("databaseName")
 
-        database_name = self.service_connection.__dict__.get(
-            "database", custom_database_name or "default"
-        )
+        database_name = self.service_connection.__dict__.get("database")
```

Comment:
this will always return None as we have removed it from schema itself

Reply:
these is in `common_db_source.py` so other database might have database in their jsonSchema

Reply:
oh okay I thought we are doing it for druid/metadata but yes in case of  `common_db_source` it makes sense but I believe the previous implementation should work then right ?


```
database_name = self.service_connection.__dict__.get("database", custom_database_name or "default")
```

Reply:
yes